.github/workflows/ci.yml

```diff
@@ -170,180 +80,139 @@ jobs:
-      FLUVIO_CMD: true
-      FLV_SOCKET_WAIT: 600
-      FLV_TEST_CONSUMER_WAIT: 120000
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Install sccache
-        run: brew install sccache
-      - name: Install ${{ matrix.rust }}
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{ matrix.rust }}
-          override: true
-      - name: Cache cargo registry
-        uses: actions/cache@v2
-        continue-on-error: true
-        with:
-          path: ~/.cargo/registry/cache
-          key: ${{ runner.os }}-${{ matrix.conf }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.conf }}-cargo-registry-
-      - name: Cache sccache output
-        uses: actions/cache@v2
-        continue-on-error: true
-        with:
-          path: /Users/runner/Library/Caches/Mozilla.sccache
-          key: ${{ runner.os }}-${{ matrix.conf }}-sccache-${{ hashFiles('**/Cargo.*') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.conf }}-sccache-
-      - name: Start sccache server
-        run: sccache --start-server
-      - name: Build Test
-        run: |
-          make RELEASE=release build_test
-      - name: Install Helm
-        run: actions/ci-replace-helm.sh
-        env:
-          HELM_VERSION: v3.3.4
-          OS: ${{ matrix.os }}
-      - run: helm version
-      - name: Install Minikube on Mac
-        run: brew install minikube
-      - name: Setup Minikube for Mac
-        timeout-minutes: 10
-        run: |
-          minikube start --driver=virtualbox --kubernetes-version 1.19.6
-      - name: Test minikube
-        run: |
-          minikube profile list
-          minikube status
-      - name: Setup installation pre-requisites
-        run: |
-          ./target/release/fluvio cluster start --setup --local --develop
-      # - name: Setup tmate session
-      #  uses: mxschmitt/action-tmate@v3
-      - name: Print Fluvio version
-        run: |
-          ./target/release/fluvio version
-      - name: smoke test tls
-        timeout-minutes: 2
-        run: |
-          make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
-      - name: Build examples
-        run: examples/tests/build.sh --release
-      - name: Test examples
-        timeout-minutes: 1
-        run: examples/tests/run.sh --release
-      - name: Stop sccache server
-        run: sccache --stop-server
-      - name: Upload Sc logs
-        timeout-minutes: 5
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          name: fluvio-local-sc-logs
-          path: /usr/local/var/log/fluvio/flv_sc.log
-      - name: Upload Spu logs
-        timeout-minutes: 5
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          name: fluvio-local-spu-logs
-          path: /usr/local/var/log/fluvio/spu_log_5001.log
+#  mac_local_cluster_test:
```

Comment:
let's remove commented out code

Reply:
Sure... the only reason I hadn't removed this was because it had specific setup that I didn't re-capture anywhere else. Actually, I may be able to incorporate it into the other local cluster test now that I know how to use the matrix functionality better. But I'll remove it for now, if we want to get it back we can dig in git.